src/views/sessions/OnlineSession/OnlineSessionList.vue

```diff
@@ -9,7 +9,7 @@
 </template>
 
 <script>
-import GenericListPage from '@/layout/components/GenericListPage'
+import { GenericListPage } from '@/layout/components'
 import { DetailFormatter } from '@/components/Table/TableFormatters'
 
 export default {
```

Comment:
I'm sorry, but I do not have information about the specific code you've mentioned as it does not provide enough context. Could you please specify which codebase or file(s) you want me to analyze?